### PR TITLE
Julie/seo indexhiddenpages docs

### PR DESCRIPTION
**Pull request by juliesaia**
Add docs for seo.indexHiddenPages https://github.com/mintlify/mint/pull/1244

### CODE REVIEW

**Review by hahnbeelee — CHANGES_REQUESTED** [review]
Can you also add it to https://mintlify.com/docs/settings/global ?

File: settings/global.mdx [hunk 1]
```diff
@@ -375,7 +375,7 @@ Example:
 
 ### Redirects
 
-<ResponseField name="redirects" type="Redirect[]" required>
+<ResponseField name="redirects" type="Redirect[]">
```
Comment (juliesaia) [inline settings/global.mdx:378]: this shouldnt have been required i think lol

Comment (hahnbeelee) [reply settings/global.mdx:378]: lol i agree

**Review by hahnbeelee — APPROVED** [review]
LGTM 🚢

**Pull request merged**